src/lib.rs

```diff
@@ -177,9 +179,9 @@ mod test {
         }
 
         is_send::<DB>();
-        is_send::<DBIterator<'_>>();
-        is_send::<DBRawIterator<'_>>();
-        is_send::<Snapshot>();
+        is_send::<DBIterator<'_, DB>>();
+        is_send::<DBRawIterator<'_, DB>>();
+        is_send::<Snapshot<DB>>();
```

Comment:
Hmm, come to think of it, I better off type-aliasing these like `DB` for maximum compatibility?

Reply:
done